### PR TITLE
Drop CoffeeScript from Gulp tasks

### PR DESCRIPTION
**Pull request by lipis**
### Not working
- [ ] `gulp build`

### CODE REVIEW

**Comment by rbubley** [comment]
The travis logs seem to show a race condition where `copy_bower_files` is running before `bower install`.

**Comment by lipis** [comment]
Any idea why?

**Comment by lipis** [comment]
@gmist?

File: gulp/tasks/dep.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    {
+      cmd: cmd,
+      match: /bower.json$/,
+    },
+  ];
+  gulp
+    .src('bower.json')
+    .pipe($.plumber())
+    .pipe($.start(start_map));
+});
```
Comment (rbubley) [inline gulp/tasks/dep.js]: The gulp docs on start are missing, but I suspect re-writing this gulp task so as to not use `start` will fix the problem.

Comment (lipis) [reply gulp/tasks/dep.js]: can you submit a PR?

Comment (rbubley) [reply gulp/tasks/dep.js]: Sorry, not on this one: I've never properly got to grip with `gulp`.

**Comment by rbubley** [comment]
Think you can close this now.

**Pull request closed without merging**